modules/activity_tracker/variables.tf

```diff
@@ -152,6 +152,17 @@ variable "log_analysis_targets" {
   sensitive   = true
 }
 
+# Cloud Logs Targets
+variable "cloud_logs_targets" {
+  type = list(object({
+    instance_id   = string
+    target_region = optional(string)
+    target_name   = string
```

Comment:
You should probably add one more option in here, and thats the ability to skip the 2s2 auth policy creation. Make it optional and default it to false (do no skip).
Then update the logic in the `ibm_iam_authorization_policy` resource to only create if skip is set to false